TopAnalysis/interface/TTbarDoubleLeptonCppWorker.h

```diff
@@ -83,7 +83,7 @@ class TTbarDoubleLeptonCppWorker {
   float out_Jets_CSVv2[maxNGoodJetsToKeep_];
 
   const static unsigned short maxNCutStep = 100;
-  const unsigned short out_nCutStep;
+  int out_nCutStep;
```

Comment:
Let us just keep cut step. then you can remove maxNCutStep and rename nCutStep->CutStep.
int should be unsigned short.